esx_service/vmdk_ops.py

```diff
@@ -658,7 +658,9 @@ def reset_vol_meta(vmdk_path):
     vol_meta[kv.ATTACHED_VM_UUID] = None
     vol_meta[kv.ATTACHED_VM_NAME] = None
     if not kv.setAll(vmdk_path, vol_meta):
-       logging.warning("clear_volume_status: Failed to save Disk metadata for %s", vmdk_path)
+       logging.warning("reset_vol_meta: Failed to save Disk metadata for %s", vmdk_path)
+       msg = "Failed to reset stale metadata for {0}.".format(vmdk_path)
```

Comment:
I think you should use the same message for warning and return message.  Any special reason they are different ?